src/zcash/Note.cpp

```diff
@@ -204,14 +205,36 @@ boost::optional<SaplingNotePlaintext> SaplingNotePlaintext::decrypt(
 
     assert(ss.size() == 0);
 
+    uint256 pk_d;
+    if (!librustzcash_ivk_to_pkd(ivk.begin(), ret.d.data(), pk_d.begin())) {
+        return boost::none;
+    }
+
+    uint256 cmu_expected;
+    if (!librustzcash_sapling_compute_cm(
```

Comment:
This doesn't compute `cm`, it computes `cmu`. I know renaming it is awkward but please file a ticket to do that.

Reply:
https://github.com/zcash/zcash/issues/3446